lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetTypeMapping.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.parquet.writer;
+
+import io.trino.spi.type.Type;
+
+import java.util.OptionalInt;
+
+import static java.util.Objects.requireNonNull;
+
+public record ParquetTypeMapping(OptionalInt id, Type type, String name)
```

Comment:
nit: consider `ParquetTypeMapping(OptionalInt id, String name, Type type)`

Reply:
outdated due to removed ParquetTypeMapping